planning/behavior_velocity_out_of_lane_module/src/filter_predicted_objects.cpp

```diff
@@ -85,7 +85,13 @@ void cut_predicted_path_beyond_red_lights(
   const PlannerData & planner_data, const double object_front_overhang)
 {
   const auto stop_line = find_next_stop_line(predicted_path, planner_data);
-  if (stop_line) cut_predicted_path_beyond_line(predicted_path, *stop_line, object_front_overhang);
+  if (stop_line) {
+    auto longer_stop_line = *stop_line;
+    const auto diff = stop_line->back() - stop_line->front();
+    longer_stop_line.front() += diff * -2.0;
+    longer_stop_line.back() += diff * 2.0;
```

Comment:
Does this mean that the stop line width becomes 5 times the original one? Isn't it too wide?
If you think it's fine, I'm okay.

Reply:
You are right I should not need to make it this big but it does not work as expected with smaller values :thinking: I will check what is the issue.

Reply:
I fixed the issue and now I only make the stop line twice bigger.